[12:11:38] {main.py:18} INFO - ___________NEW RUN STARTING___________
[12:11:38] {main.py:19} INFO - Code started at: 2022-01-22 12:11:38.422169
[12:11:38] {/opt/virtualenvs/python3/lib/python3.8/site-packages/urllib3/connectionpool.py:971} DEBUG - Starting new HTTPS connection (1): dumbot.drdum.repl.co:443
[12:11:39] {/opt/virtualenvs/python3/lib/python3.8/site-packages/urllib3/connectionpool.py:452} DEBUG - https://dumbot.drdum.repl.co:443 "GET / HTTP/1.1" 200 17
[12:11:39] {/home/runner/DumBot/keep_alive.py:27} INFO - Status code: 200
[12:11:39] {/opt/virtualenvs/python3/lib/python3.8/site-packages/urllib3/util/retry.py:333} DEBUG - Converted retries value: 3 -> Retry(total=3, connect=None, read=None, redirect=None, status=None)
[12:11:39] {/opt/virtualenvs/python3/lib/python3.8/site-packages/google/auth/transport/requests.py:177} DEBUG - Making request: POST https://oauth2.googleapis.com/token
[12:11:39] {/opt/virtualenvs/python3/lib/python3.8/site-packages/urllib3/connectionpool.py:971} DEBUG - Starting new HTTPS connection (1): oauth2.googleapis.com:443
[12:11:39] {/opt/virtualenvs/python3/lib/python3.8/site-packages/urllib3/connectionpool.py:452} DEBUG - https://oauth2.googleapis.com:443 "POST /token HTTP/1.1" 200 None
[12:11:39] {/opt/virtualenvs/python3/lib/python3.8/site-packages/urllib3/connectionpool.py:971} DEBUG - Starting new HTTPS connection (1): www.googleapis.com:443
[12:11:39] {/opt/virtualenvs/python3/lib/python3.8/site-packages/urllib3/connectionpool.py:452} DEBUG - https://www.googleapis.com:443 "GET /drive/v3/files?q=mimeType%3D%22application%2Fvnd.google-apps.spreadsheet%22+and+name+%3D+%22BotCommands%22&pageSize=1000&supportsAllDrives=True&includeItemsFromAllDrives=True&fields=kind%2CnextPageToken%2Cfiles%28id%2Cname%2CcreatedTime%2CmodifiedTime%29 HTTP/1.1" 200 None
[12:11:39] {/opt/virtualenvs/python3/lib/python3.8/site-packages/urllib3/connectionpool.py:971} DEBUG - Starting new HTTPS connection (1): sheets.googleapis.com:443
